guides/common/assembly_preparing-environment-for-capsule-installation.adoc

```diff
@@ -2,6 +2,13 @@ ifdef::context[:parent-context: {context}]
 
 [id="preparing-environment-for-capsule-installation"]
 = Preparing your environment for installation
+
+Review prerequisites below before you install {SmartProxyServer}.
```

Comment:
This new line is based on https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/con_upgrading-overview.adoc